br/backup-and-restore-overview.md

```diff
@@ -156,7 +156,7 @@ The following table lists the compatibility matrix for full backups:
 | v7.5.0    | v7.5.0 and later | -                 |
 | v8.1.0    | v8.1.0 and later | -                 |
 
-The following table lists the compatibility matrix for log backups:
+The following table lists the compatibility matrix for log backups. Note that all data in the table comes from newly created clusters.
```

Comment:
for clusters upgraded from version prior to v7.2.0 to v7.2.0 or above, their behavior is as of backup from v7.1.0.